.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,9 @@
+# Contributing to GHDL
+
+GHDL is a free and open source software brought to you with :heart: by [@tgingold](https://github.com/tgingold) and [contributors](https://github.com/ghdl/ghdl/graphs/contributors). So, first off, thanks for taking the time to contribute. We need all the help we can get :thumbsup:!
```

Comment:
GitHub should recognize @tgingold without the need to write a full link `[@tgingold](https://github.com/tgingold)` or is the purpose to create a blue link instead of a black link?

Reply:
I think that usernames are only recognized in the body of issues and comments, but not in every markdown file. For example ghdl/ghdl#582 works here, but not in the `README.md`. Contributing is expected to be rendered as `README.md`, not as `bug_report.md`.